README.md

```diff
@@ -1,41 +1,54 @@
 # Welcome to the scheduled events lab!
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
 
-## About the codebase
+This lab is an exercise to check your knowledge on scheduled events. It is automatically graded via a workflow once you have completed the instructions.
+
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
+
+## About this lab
+
+:warning: A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and **altering their contents will affect the repository's ability to assess your lab and give feedback.**
+
+:warning: This lab utilizes [GitHub Actions](https://docs.github.com/en/actions), which is free for public repositories and self-hosted runners, but may incur charges on private repositories. See *[About billing for GitHub Actions]* to learn more.
+
+:information_source: The use of GitHub actions also means that it may take the grading workflow a few seconds and sometimes minutes to run.
+
+## Instructions
+
 In the codebase for this repo, you'll find 3 workflows which we describe below with their intended purpose. Each workflow is complete **except for** the `on:` key.
-1. `stale-daily.yaml`: checks for stale issues daily at midnight and closes them
-2. `stale-weekly.yaml`: checks for stale issues every Monday and closes them
-3. `stale-monthly.yaml`: checks for stale issues on the first of every month and closes them
 
-A grading script exists at `.github/grading.yml`. You do not need to use this workflows for any purpose and altering its contents will affect the repository's ability to assess your lab and give feedback.
+- `stale-daily.yaml`: checks for stale issues daily at midnight and closes them
+- `stale-weekly.yaml`: checks for stale issues every Monday and closes them
+- `stale-monthly.yaml`: checks for stale issues on the first of every month and closes them
```

Comment:
Similarly, we can probably make this more clear by using full paths: `.github/workflows/sale-daily.yml`. Alternatively, we can just mention above this section that these files are all in `.github/workflows/`

I also made a mistake in these file names! Extensions are all `.yml` not `.yaml`

Reply:
Ok, will correct!